cmd/create.go

```diff
@@ -182,9 +182,9 @@ A full list of component types that can be deployed is available using: 'odo cat
 }
 
 func init() {
-	componentCreateCmd.Flags().StringVar(&componentBinary, "binary", "", "Use a binary as the source file for the component")
-	componentCreateCmd.Flags().StringVar(&componentGit, "git", "", "Use a git repository as the source file for the component")
-	componentCreateCmd.Flags().StringVar(&componentLocal, "local", "", "Use local directory as a source file for the component")
+	componentCreateCmd.Flags().StringVar(&componentBinary, "binary", "b", "Use a binary as the source file for the component")
```

Comment:
Doesn't this mean the default value for `binary` flag is `b`?
Shouldn't we use StringVarP instead?
Please correct me if I am wrong